src/components/hypercube-table.jsx

```diff
@@ -8,29 +8,31 @@ import useResolvedValue from './use/resolved-value';
 
 import './hypercube-table.pcss';
 
-function cellGetterForIndex(index) {
+function cellGetterForIndex(index, layout) {
   return function cellGetter({ rowData }) {
-    if (!rowData || !rowData[index]) {
+    const indexInQMatrix = layout.qHyperCube.qEffectiveInterColumnSortOrder.indexOf(index);
```

Comment:
Can we move this out to the outer close as this gets called for every cell rendered and the indexInQMatrix won't change?

Reply:
ah, of course :)